scheduler.lisp

```diff
@@ -4,6 +4,7 @@
   (:use)
   (:export #:scheduler
            #:in-memory-scheduler
+           #:sqlite-scheduler
```

Comment:
I remember that I have suggested that, but I don't remember why. Why do we export the `sqlite-scheduler` from the `scheduler` package? If someone knows that they want to use the sqlite-scheduler, then they should be aware of scheduler-sqlite package. Please remove the export from `scheduler.lisp`.

Reply:
Done